gobblin-modules/gobblin-orc/src/main/java/org/apache/gobblin/writer/GenericRecordToOrcValueWriter.java

```diff
@@ -344,7 +346,9 @@ public void addValue(int rowId, int column, Object data, ColumnVector output) {
       // make sure the child is big enough
       if (cv.childCount > cv.keys.isNull.length) {
         int resizedLength = resize(rowsAdded, cv.isNull.length, cv.childCount);
+        log.info("Column vector: {}, resizing to: {}, child count: {}", cv.keys, resizedLength, cv.childCount);
         cv.keys.ensureSize(resizedLength, true);
+        log.info("Column vector: {}, resizing to: {}, child count: {}", cv.values, resizedLength, cv.childCount);
```

Comment:
just double checking: do the `.keys` and `.values` column vectors have a meaningful `.toString`... or were you wanting actually to show their `.isNull.length`?

they likely grow in lockstep, but is it intentional to show the keys before the potential resize, but show the values after?

Reply:
I actually want their identity hash values. ORC will not have a meaningful tostring which includes data as that can take a lot of overhead to display

Reply:
suggestion: if all you need is a hash, then perhaps writing and reusing hashing function in Gobblin Util can help?

Reply:
I'm looking to log something like this 
```Column vector: org.apache.orc.storage.ql.exec.vector.BytesColumnVector@6a4c0a7f, resizing to: 257610, child count: 25761``` where I need both the identityHash and the referring class name.

I can look into the Util class. Can u point me to it? But IMO might be an overkill in replacing `cv`  to `GobblinUtil.hash(cv)` , no?

Reply:
Re-read, since you just need uniquely identifiable hash, current approach seems ok.